demisto_sdk/commands/validate/validators/RM_validators/RM109_is_readme_exists.py

```diff
@@ -0,0 +1,45 @@
+
+from __future__ import annotations
+
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.common.tools import get_pack_name
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Pack, Playbook]
+
+
+class IsReadmeExistsValidator(BaseValidator[ContentTypes]):
+    error_code = "RM109"
+    description = "Validates if there is a readme file for the content item."
+    rationale = "Insure that we have a README file for uploading to xsoar.pan.dev"
+    error_message = "There is no README file for content item from type {0} in pack named '{1}'. Please add relevant README."
```

Comment:
```suggestion
    error_message = "The {0} doesn't have a README file. Please add a README.md file in the content item's directory."
```
Where 0 is the content item type.